proxy/proxies/nginx/default.go

```diff
@@ -0,0 +1,96 @@
+package nginx
+
+import (
+	log "github.com/zoumo/logdog"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/pkg/api/v1"
+	extensions "k8s.io/client-go/pkg/apis/extensions/v1beta1"
+)
+
+const (
+	defaultHTTPBackendImage     = "cargo.caicloud.io/caicloud/default-http-backend:v0.1.0"
```

Comment:
you can specify the image with cmd flag --default-http-backend, it will be stored in defaultHTTPbackend of nginx struct
See nginx.AddFlags() function